test/bigdecimal/test_bigdecimal.rb

```diff
@@ -1660,8 +1660,7 @@ def test_limit
     BigDecimal.save_limit do
       BigDecimal.limit(1)
       x = BigDecimal("3")
-      assert_equal(90, x ** 4) # OK? must it be 80?
-      # 3 * 3 * 3 * 3 = 10 * 3 * 3 = 30 * 3 = 90 ???
```

Comment:
Before:  `3**4` → `(3 * (3**2)) * 3`  → `(3 * 9) * 3` → `round(27) * 3` → `90`
After: `3**4` → `(3 * 3) * (3**2)` → `9 * 9` → `round(81)` → `80`

Although this test case seems to be improved, just a coincidence.
This pull request is not a bug fix. Large error still exist.
```ruby
BigDecimal.limit 1
(BigDecimal('2')**100).to_i / 2**100 #=> 1577
```